docs/source/custom_scoring.md

```diff
@@ -9,23 +9,28 @@ In this guide, we will walk through the process of evaluating LLM performance us
 
 ## Define a custom scorer
 
-First make sure `nltk` is installed as a package to your environment, which our custom scorer uses.
+To create a custom scorer that satisfies the Scorer interface (defined in the section below), implement the scoring logic in the `run_batch` method. Additionally, provide your scorer a name in the `name()` method. 
 
-To create a custom scorer, we will implement the `name` and `run_batch` methods, and optionally override the `requires_reference` method if our scorer doesn't require reference or target data.
+This example custom scorer is called `TrigramRepitition`, which scores responses with a 0.0 if they contain repeated trigrams above a thresholded number of times. For our scorer, we override the `requires_reference()` method to return `False` instead of `True`, since this custom scorer evaluates the candidate outputs without the need for a reference.
 
-This example custom scorer is called `TrigramRepitition`, which scores responses with a 0.0 if they contain repeated trigrams above a thresholded number of times.
+Make sure `nltk` is installed as a package to your environment, which our custom scorer uses.
+
+```
+pip install nltk
+```
 
 ```python
 from arthur_bench.scoring import Scorer
+import nltk
+# make sure corpus is downloaded
+nltk.download('punkt')
 from nltk import trigrams
 from typing import List, Optional
 
-# make sure corpus is downloaded
-nltk.download('punkt')
 
 class TrigramRepetition(Scorer):
 
-    def __init__(self, threshold: int = 5):
+    def __init__(self, threshold: int = 3):
```

Comment:
won't changing this to three change the expected outputs at the end?

Reply:
yes, the expected outputs at the end are [1.0, 0.0], which this works for

the old example was not scoring the repeated one as a 0.0 because it had less than 5!!